admin/spam/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from ckeditor.widgets import CKEditorWidget
+
+
+class EmailForm(forms.Form):
+    author = forms.CharField(label='Author', max_length=100)
+    email = forms.ChoiceField(label='Email')
+    subject = forms.CharField(label='Subject', required=True)
+    message = forms.CharField(label='Message', required=True,
+                              widget=CKEditorWidget())
+
+    def __init__(self, *args, **kwargs):
+        choices = kwargs.get('initial', {}).get('email', [])
+        self.base_fields['email'] = forms.ChoiceField(choices=choices)
```

Comment:
`self` in the last line makes no sense since self is not defined outside of functions.